repos/system_upgrade/el7toel8/actors/quaggatofrr/tests/test_unit_quaggatofrr.py

```diff
@@ -0,0 +1,79 @@
+import contextlib
+import os
+import shutil
+
+from leapp.libraries.actor import quaggatofrr
+
+ACTIVE_DAEMONS = ['bgpd', 'ospfd', 'zebra']
+CUR_DIR = os.path.dirname(os.path.abspath(__file__))
+FROM_DIR = '/tmp/from_dir/'
+TO_DIR = '/tmp/to_dir/'
+CONFIG_DATA = {
+    'bgpd': '--daemon -A 10.10.100.1',
+    'isisd': '--daemon -A ::1',
+    'ospf6d': '-A ::1',
+    'ospfd': '-A 127.0.0.1',
+    'ripd': '-A 127.0.0.1',
+    'ripngd': '-A ::1',
+    'zebra': '-s 90000000 --daemon -A 127.0.0.1'
+}
+
+
+@contextlib.contextmanager
+def _create_mock_files():
+    try:
+        os.mkdir(FROM_DIR)
+        os.mkdir(TO_DIR)
+
+        for num in range(1, 10):
+            full_path = "{}test_file_{}".format(FROM_DIR, num)
+            with open(full_path, 'w') as fp:
+                fp.write("test_file_{}".format(num))
+        yield
+    finally:
+        shutil.rmtree(FROM_DIR)
+        shutil.rmtree(TO_DIR)
```

Comment:
No action required, just FYI:

@mruprich it is safer implementation now. If someone in future will use this function, it is a chance he will forget to cleanup.  So it is usually better to implement teardown via context managers or fixtures with the context managers (https://docs.pytest.org/en/stable/fixture.html#fixture-finalization-executing-teardown-code)

It is also worse to mention that pytest provides you a fixture to create temp dirs (https://docs.pytest.org/en/stable/tmpdir.html), but it is just FYI as well